test/images/agnhost/porter/porter.go

```diff
@@ -19,20 +19,32 @@ limitations under the License.
 // Reads env vars; for every var of the form SERVE_PORT_X, where X is a valid
 // port number, porter starts an HTTP server which serves the env var's value
 // in response to any query.
-package main
+
+package porter
 
 import (
 	"fmt"
 	"log"
 	"net/http"
 	"os"
 	"strings"
+
+	"github.com/spf13/cobra"
 )
 
 const prefix = "SERVE_PORT_"
 const tlsPrefix = "SERVE_TLS_PORT_"
 
-func main() {
+// CmdPorter is used by agnhost Cobra.
+var CmdPorter = &cobra.Command{
+	Use:   "porter",
+	Short: "TBA",
+	Long:  "TBA",
```

Comment:
Just calling out the remaining TBA

Reply:
Good point, I was glad I finished with those. :D